tests/data/RGB.byte.tif.aux.xml

```diff
@@ -0,0 +1,29 @@
+<PAMDataset>
```

Comment:
Added this file to cover up outdated and erroneous stats stored in the RGB.byte.tif file, which GDAL can't directly erase as far as I know.